src/state_mgmt/state/state.gd

```diff
@@ -1,5 +1,5 @@
 class_name FrayState
-extends RefCounted
+extends Resource
```

Comment:
FrayState becoming a Resource is interesting. Saving and referencing other state machine should be easier now with `Resource.load`. Great change

Reply:
Transitions will also become a Resource. I had meant to include the change in this commit but I just noticed while testing that it slipped my mind. Will be pushing a fix shortly.